Cargo.toml

```diff
@@ -23,6 +23,7 @@ nameattr = ["indexmap"]
 
 [dependencies]
 ahash = "0.8"
+anyhow = "1.0.70"
```

Comment:
Why do we now need a dependency on `anyhow`?

Reply:
Since there are some common functions which can emit error. I thought anyhow can make the error output more informative easility(rather than cascading`return invalid_data_error!(...)`). 

e.g. When there is no ref id in a <backtrace/> tag, we can get a error like:
```
Get ref id of backtrace failed.

Caused by:
    No ref id found
```
just by adding two `.context(` in parsing code.

The error type is eventually transformed into io error `InvalidData`, just like other collapser does.

However if there are some limitation on dependency introducing, I'll remove it.

Reply:
Anyhow is nice in general, but I'd like to avoid adding it in this particular PR :)

Reply:
Removed